plugins/parsers/grok/influx_patterns.go

```diff
@@ -13,6 +13,7 @@ const DEFAULT_PATTERNS = `
 #   https://www.elastic.co/guide/en/logstash/current/plugins-filters-grok.html
 # All default logstash patterns are supported, these can be viewed here:
 #   https://github.com/logstash-plugins/logstash-patterns-core/blob/master/patterns/grok-patterns
+# When using default logstash patterns, a semantic name is required.
```

Comment:
I'm not sure anyone would ever see this file, in fact maybe we should remove this comment altogether?  We could add it to https://github.com/influxdata/telegraf/blob/master/plugins/inputs/logparser/README.md and https://github.com/influxdata/telegraf/blob/master/plugins/parsers/grok/README.md.

That said, I don't think this is quite right.  A semantic name is only required if you wish to capture the pattern.  Perhaps in the paragraph above we should say something along the lines of "If a pattern does not have a semantic name it will not be captured".